vl-proza-message.src.js

```diff
@@ -136,6 +154,76 @@ export class VlProzaMessage extends VlElement(HTMLElement) {
             this._element.appendChild(this._getEditButtonTemplate());
         }
     }
+
+    __initWysiwyg() {
+        this.__unwrapWysiwygElement();
+        tinyMCE.baseURL = '/node_modules/tinymce';
+        tinyMCE.init({
+            target: this._wysiwygElement,
+            menubar: false,
+            inline: true,
+            toolbar: false,
+            plugins: ['quickbars'],
+            quickbars_selection_toolbar: 'bold italic underline',
+            powerpaste_word_import: 'clean',
+            powerpaste_html_import: 'clean',
+            content_css: '/style.css',
+            verify_html: false,
```

Comment:
k weet nie exact wat dees wilt zeggen maar ik zou daar precies liever true zien staan?

Reply:
`This option enables or disables the element cleanup functionality. If you set this option to false, all element cleanup will be skipped but other cleanup functionality such as URL conversion will still be executed.`

Wanneer dit aanstaat gaat TinyMCE de HTML valideren en bijgevolg ook manipuleren. Zo ontstonden er bijvoorbeeld <p>&nbsp;</p> tags.